### PR TITLE
[STATION] fix default qslmsg empty when duplicate station

### PR DESCRIPTION
**Pull request by abarrau**
hello, 
fix copy eqsl_default_qslmsg when duplicate station

tested : 
- duplicate station with empty default qslmsg = ok
- duplicate station with value on default qslmsg = ok
- duplicate station with value on default qslmsg and change with new value = ok

### CODE REVIEW

File: application/controllers/Station.php [hunk 1]
```diff
@@ -119,12 +119,23 @@ public function copy($id)
 			$data['my_station_profile']->station_profile_name = '';
 
 			if ($this->form_validation->run() == FALSE) {
+				// [eQSL default msg] GET from user options (same as edit()) //
+				$this->load->model('user_options_model');
```
Comment (HB9HIL) [inline application/controllers/Station.php:123]: you don't need to load the model, as it is in the autoloader
```suggestion
```

File: application/controllers/Station.php [hunk 2]
```diff
@@ -119,12 +119,23 @@ public function copy($id)
...
 				$this->load->view('station_profile/edit');
 				$this->load->view('interface_assets/footer');
 			} else {
-				$this->stations->add();
-
+				if (($station_id = $this->stations->add()) !== false) {
+					// [eQSL default msg] ADD to user options (same as create()) //
+					$eqsl_default_qslmsg = xss_clean($this->input->post('eqsl_default_qslmsg', true));
+					if (!empty(trim($eqsl_default_qslmsg))) {
+						$this->load->model('user_options_model');
```
Comment (HB9HIL) [inline application/controllers/Station.php:135]: no need to load the model here

```suggestion
```

**Review by HB9HIL — COMMENTED** [review]
imho we don't need the last function in this file. 
```
public function get_options()
	{
```
You wrote this 3 months ago, but as far I can see it is not used and just a redundant function of the main user_options function.

**Comment by int2001** [comment]
Thank you, @abarrau - your Fix is working.
But we need that within the station_model, since it belongs to the station_profile.
I am already trying it to implement there.

File: application/controllers/Station.php (hunk 1 above)
Comment (abarrau) [reply application/controllers/Station.php]: i delete this load model on create, update and copy

**Comment by abarrau** [comment]
> But we need that within the station_model, since it belongs to the station_profile.
> I am already trying it to implement there.

this function is used by js call on qso page for refresh qslmsg texteara, so this function is used

**Comment by abarrau** [comment]
Tested : 
- create, update, copy station profil
- update msgqsl from qso page

**Comment by abarrau** [comment]
> But we need that within the station_model, since it belongs to the station_profile.

@int2001, can i help about something ? 
you prefered i put the get_options() on station model ?

**Comment by int2001** [comment]
i show you when finished. it's a little tricky and to complex to describe.

**Comment by int2001** [comment]
you're right. whoever putted that into autoload (tbh: I am not a friend of autoload). Thanks for the finding, will fix

**Comment by abarrau** [comment]
close this (reported on 175)

**Pull request closed without merging**